### ISSUE TITLE
Playtest questions for prototype 2

### ISSUE THREAD

**Issue by Naomi-Ifergan** [Naomi-Ifergan/prototype-2#1]
1. What’s your initial thought on the game?
2. Is the game hard to play without any instructions? Explain why.
3. Is the speed of the game too fast or slow?
4. How does the aesthetic of the game make you feel?
5. How many tries did it take for you to understand the game and why?
6. Did you enjoy the game? Was it frustrating, entertaining, challenging…?

(After answering these questions, read the Wiki if you're curious about how the game is supposed to be then write whether or not the game makes more sense to you now).

**Comment by PatrickChavez** [comment]
**1. What’s your initial thought on the game?**
It was quite trippy and experimental. The visual effects were cool.

**2. Is the game hard to play without any instructions? Explain why.**
Yes, although my issues are probably due to the fact that I played this prototype with Unity 2020.2.2f1: at first I couldn’t move due to the PlayerMovement script being assigned to the Main Camera. Once I fixed the issue, I already knew the controls thanks to the script. Despite this, I wasn’t sure what the goal of the game was. I also found it strange that I couldn’t move after hitting a wall.

**3. Is the speed of the game too fast or slow?**
I would say that the game speed is fine. There was usually enough time for me to realize what was up ahead and restarting wasn’t frustrating because it only took a couple seconds to reach the part where I “died”/hit a wall.

**4. How does the aesthetic of the game make you feel?**
It was disorienting at first, but I quickly got used to it. The constantly multiplying walls make things seem larger than they are. The visual effects also convey a sense of speed.  

**5. How many tries did it take for you to understand the game and why?**
Explained in questions 2 and 3.

**6. Did you enjoy the game? Was it frustrating, entertaining, challenging…?**
Yes, it was quite entertaining and if a respawn system were to be implemented (where the game would reset once the player hit a wall), then I think that would improve the overall pacing.

**(After answering these questions, read the Wiki if you're curious about how the game is supposed to be then write whether or not the game makes more sense to you now).**
There is nothing in the wiki at the time of writing.

**Comment by LiliaIsABell** [comment]
**1. What’s your initial thought on the game?**
I thought it was very fun. It reminded me of a game I played as a kid: "Move it or Luge it".
**2. Is the game hard to play without any instructions? Explain why.**
A little bit because I wasn't sure what keys to use and what my goal was.
**3. Is the speed of the game too fast or slow?**
It is definitely fast but I liked it that way.
**4. How does the aesthetic of the game make you feel?**
I found it impressive, but then it got distracting and disorienting since everything was green.
**5. How many tries did it take for you to understand the game and why?**
It took me two tries because the first was, of course, my first impression and by the second try I found out I needed to used ASWD keys (after reading the Wiki, I realized I only needed AD) to move and make it to the end.
**6. Did you enjoy the game? Was it frustrating, entertaining, challenging…?**
I enjoyed it. I liked the challenge and really wanted to succeed, even if I didn't know what success was, and I liked how dynamic it looked.

**Comment by LeVieuxSinge** [comment]
**What’s your initial thought on the game?**
I had the feeling I would need to dodge or use the walls to proceed. Visually it felt empty but simple.

**Is the game hard to play without any instructions? Explain why.**
The controls were intuitive. A and D are common movement keys so it's always safe to use.

**Is the speed of the game too fast or slow?**
The start is fine but it becomes speedy quite fast. Using the walls to slow yourself down makes the game more tolerable.

**How does the aesthetic of the game make you feel?**
The aesthetic was confusing. I'm guessing the "glitchy" effect was intentional but I think it was a bit too much, to a point where I couldn't even differentiate the platform from the void.

**How many tries did it take for you to understand the game and why?**
2 tries.

**Did you enjoy the game? Was it frustrating, entertaining, challenging…?**
The game has the potential to be enjoyable and addictive but it still needs more content, especially in terms of length.

**Comment by CocoChao** [comment]
**•	What is your initial thought on the game?**
I found it very glitchy because the walls were duplicated as I move forward. It was not until I read your prototype wiki and found out that it is an issue you are trying to fix. Also, I was not sure what I was doing at first because the right and left motion were not obvious.

**•	Is the game hard to play without any instructions? Explain why.**
Yes. The game started quickly, and I took me a few seconds to realize that I was moving forward. I thought if I touch the wall that it would be the end, but I could still move left and right (unless one of the cube’s faces was completely stuck on a wall).

**•	Is the speed of the game too fast or slow?**
The game was too fast, and because of the glitch, I did not know when it ends (it ended too fast whenever the cube landed flat on a wall). 

**•	How does the aesthetic of the game make you feel?**
The game was disorienting, all the green walls looked the same. There is not much to say because it was a simple and minimal aesthetic. However, I suggest you space out the walls and add different colors to each wall to create a more dynamic game.

**•	How many tries did it take for you to understand the game and why?**
It took me around 5 tries, because I wanted to know how the cube reacts when it touches the wall and when it does not.

**•	Did you enjoy the game? Was it frustrating, entertaining, challenging…? (After answering these questions, read the Wiki if you're curious about how the game is supposed to be then write whether or not the game makes more sense to you now).** 
The game was a bit confusing and frustrating. My cube would be stuck in front of the walls many times unless it is slightly rotated. It would be nice to have a points counter, to know how many walls I avoided or how many lives I have left.
